src/osemgrep/cli_scan/Scan_CLI.ml

```diff
@@ -109,7 +109,7 @@ let default : conf =
          * not overload on large machines
          *)
         Core_runner.num_jobs = min 16 (Parmap_helpers.get_cpu_count ());
-        timeout = 2.0 (* seconds *);
+        timeout = 3.0 (* seconds *);
```

Comment:
please add a coupling: comment saying if you modify here, you need to modify constants.py and User_settings.ml